org.eclipse.corrosion.tests/src/org/eclipse/corrosion/tests/TestPerspective.java

```diff
@@ -29,8 +29,10 @@
 import org.eclipse.ui.wizards.newresource.BasicNewFileResourceWizard;
 import org.eclipse.ui.wizards.newresource.BasicNewFolderResourceWizard;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 
+@Ignore
 public class TestPerspective {
```

Comment:
If it's not relevant anymore, please just remove it.

Reply:
The test was removed.